.github/workflows/linux-cpu-tests.yml

```diff
@@ -49,7 +49,38 @@ jobs:
           pip install accelerate transformers diffusers
           python -m pytest test/models
 
+
       - name: Run CLI tests
         run: |
           pip install optimum
           python -m pytest test/cli
+
+          - name: Environment
+          run: |
+            python -m venv /opt/venv && export PATH="/opt/venv/bin:$PATH"
+            python utils/print_env.py
+
+  run_staging_tests:
```

Comment:
This needs to be isolated from the previous workflow as we're using a separate domain to run the staging tests. Refer to `_run_staging` argument for more details.